src/WebJobs.Script/BindingExtensions/ExtensionsManager.cs

```diff
@@ -185,6 +187,34 @@ private void SetupProcessEnvironment(ProcessStartInfo startInfo)
             }
 
             startInfo.EnvironmentVariables.Add("DOTNET_SKIP_FIRST_TIME_EXPERIENCE", "true");
+
+            string fallbackFolderPath = _nugetFallBackPath ?? GetLatestFallbackFolderPath(NugetFallbackFolderRootPath);
+            if (!string.IsNullOrEmpty(fallbackFolderPath))
+            {
+                startInfo.EnvironmentVariables.Add("RestoreFallbackFolders ", fallbackFolderPath);
+            }
+
+            startInfo.EnvironmentVariables.Add("NUGET_XMLDOC_MODE", "skip");
+        }
+
+        private string GetLatestFallbackFolderPath(string nugetFallbackRootFolder)
+        {
+            string directoryPath = null;
+            if (FileUtility.DirectoryExists(nugetFallbackRootFolder))
+            {
+                Version latestVersion = null;
+                Version tempVersion = null;
+                foreach (var item in FileUtility.EnumerateDirectories(nugetFallbackRootFolder))
+                {
+                    var directoryName = FileUtility.DirectoryInfoFromDirectoryName(item).Name;
+                    if (Version.TryParse(directoryName, out tempVersion) && (latestVersion == null || tempVersion > latestVersion))
+                    {
+                        latestVersion = tempVersion;
+                        directoryPath = item;
+                    }
+                }
+            }
+            return directoryPath;
```

Comment:
set _nugetFallBackPath?

Reply:
This could be simplified with mapping and sorting (some LINQ love will make this a one liner)